node/src/chain_spec/mod.rs

```diff
@@ -1,15 +1,41 @@
+mod additional_chain_spec;
 mod battery_park;
 mod dev;
 mod local_testnet;
 
+pub use additional_chain_spec::AdditionalChainSpec;
 pub use battery_park::battery_park_config;
 pub use dev::dev_config;
 pub use local_testnet::local_testnet_config;
 use sp_core::{Pair, Public};
 use sp_runtime::traits::{IdentifyAccount, Verify};
-use zeitgeist_primitives::types::{AccountId, Signature};
-use zeitgeist_runtime::{BalancesConfig, TokensConfig};
+use zeitgeist_primitives::types::{AccountId, Balance, Signature};
+use zeitgeist_runtime::TokensConfig;
 
+#[cfg(feature = "parachain")]
+use {sp_runtime::Perbill, zeitgeist_primitives::ztg};
+
+#[cfg(feature = "parachain")]
+const DEFAULT_COLLATOR_INFLATION_INFO: parachain_staking::InflationInfo<Balance> =
+    parachain_staking::InflationInfo {
+        annual: parachain_staking::Range {
+            ideal: Perbill::from_percent(ztg::STAKING),
+            max: Perbill::from_percent(ztg::STAKING),
+            min: Perbill::from_percent(ztg::STAKING),
+        },
+        expect: parachain_staking::Range {
+            ideal: ztg::COLLATORS * zeitgeist_primitives::constants::BASE,
+            min: ztg::COLLATORS * zeitgeist_primitives::constants::BASE,
+            max: ztg::COLLATORS * zeitgeist_primitives::constants::BASE,
```

Comment:
I haven't had the chance to delve in the staking pallet yet, so I'm not yet sure what configuring `max` and `min` means. What happens if less or more than `ztg::COLLATORS * zeitgeist_primitives::constants::BASE` is staked?

Reply:
Yes, it is confusing. `expect` is the amount of ZTG that will flow to collators with no less than `min` and no more than `max`, `annual` is self-descriptive and `round`  is the round inflation percentage defined in the `runtime`. Kind of redundant.
MB received a grant and these crates are still in active development so things will probably change in the near future.

Additional resources:

* https://github.com/PureStake/moonbeam/blob/d40f492f7d6ba6debd529da4179821d9a8d6d110/pallets/parachain-staking/src/inflation.rs#L98-L105.
* https://github.com/PureStake/moonbeam/blob/e73891ff1125e205e2f879f9a668b97a0b7fda07/node/src/chain_spec.rs#L207-L226